proto/cel/expr/checked.proto

```diff
@@ -0,0 +1,344 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package cel.expr;
```

Comment:
Why are we not including the tld in the main proto package name?

Reply:
The real requirement is: The proto package should match both the c++ namespace and the folders.

This is the (defacto?) standard way to reconcile that c++, python, es, etc, which do not typically include the tld in namespaces/packages. It is only go (which uses it as real dns address) and java (which shoves the rdomain in the namespace per the style guide) that use the tld.